include/cosim/system_structure.hpp

```diff
@@ -213,8 +213,25 @@ class system_structure
         full_variable_name target;
     };
 
+    /// Information about a powerbond connection. For use with the ecco algorithm only.
+    struct power_bond
+    {
+        /// The input_a variable in the bond, according to the description of the ecco algorithm.
+        full_variable_name input_a{"", ""};
+
+        /// The output_a variable in the bond, according to the description of the ecco algorithm.
+        full_variable_name output_a{"", ""};
+
+        /// The input_b variable in the bond, according to the description of the ecco algorithm.
+        full_variable_name input_b{"", ""};
+
+        /// The output_b variable in the bond, according to the description of the ecco algorithm.
+        full_variable_name output_b{"", ""};
```

Comment:
nitpicking, but I would add a default ctor to avoid passing the empty strings

Reply:
Ah, nice catch and good point. `std::string("")` is not the same as `std::string()`!

Reply:
Not sure what you're getting at exactly. Add a default ctor for `full_variable_name`? 

The current ctor just invokes std::move on each string:

```
    full_variable_name(std::string simulatorName, std::string variableName)
        : entity_name(std::move(simulatorName))
        , variable_name(std::move(variableName))
    { }
```

Reply:
I think you can make a default ctor and initialize those strings in the declaration or create a ctor without argument and initialize them in there.

Reply:
The easiest is just to create a default constructor like this:
```
full_variable_name() = default;
```